src/components/LanguageSelection/index.tsx

```diff
@@ -16,7 +16,19 @@ export interface LanguageSelectionProps {
 
 export function LanguageSelection({ assembly }: LanguageSelectionProps) {
   const [language, setLanguage] = useLanguage({ updateUrl: true });
-  const targets = Object.keys(assembly?.spec?.targets ?? {}) as Language[];
+  const assemblyTargets = Object.keys(
+    assembly?.spec?.targets ?? {}
+  ) as Language[];
+
+  const targets = assemblyTargets.reduce<Language[]>((languages, lang) => {
+    if (TEMP_SUPPORTED_LANGUAGES.includes(lang)) {
```

Comment:
kind of weird, but this just maps the "js" target to "typescript", and filters out anything else not in `TEMP_SUPPORTED_LANGUAGES`. There's no expectation for this code to last, just wanted to enable ts & python on the language bar